Application/Education/School/Type/Service.php

```diff
@@ -64,4 +64,43 @@ public function getTypeByName($Name)
 
         return (new Data($this->getBinding()))->getTypeByName($Name);
     }
+
+    /**
+     * @param TblType $tblType
+     *
+     * @return string
+     */
+    public function getSchoolTypeString(TblType $tblType){
```

Comment:
Codacy found an issue: [The method getSchoolTypeString() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)